openssl-sys/src/handwritten/evp.rs

```diff
@@ -646,3 +652,16 @@ extern "C" {
     pub fn EVP_EncodeBlock(dst: *mut c_uchar, src: *const c_uchar, src_len: c_int) -> c_int;
     pub fn EVP_DecodeBlock(dst: *mut c_uchar, src: *const c_uchar, src_len: c_int) -> c_int;
 }
+
+extern "C" {
```

Comment:
These should go in a new file `params.rs` as they're declared in `openssl/params.h` rather than evp.

Reply:
fixed!